src/console_cmds.cpp

```diff
@@ -2179,10 +2179,14 @@ DEF_CONSOLE_CMD(ConFont)
 		IConsolePrint(CC_HELP, "Manage the fonts configuration.");
 		IConsolePrint(CC_HELP, "Usage 'font'.");
 		IConsolePrint(CC_HELP, "  Print out the fonts configuration.");
+		IConsolePrint(CC_HELP, "  The \"Currently active\" configuration is the one actually in effect (after interface scaling and replacing unavailable fonts).");
+		IConsolePrint(CC_HELP, "  The \"Requested\" configuration is the one requested via console command or config file.");
 		IConsolePrint(CC_HELP, "Usage 'font [medium|small|large|mono] [<name>] [<size>] [aa|noaa]'.");
```

Comment:
Can you change "name" here to something less ambiguous. "Name of font" or "Font path" ??

Reply:
I can't think of a simple explanation here so I'm just going to try my best to layout the weird things I've learned when it comes to loading fonts.

The console command won't act on: 
"font medium 12 aa" 
if the default font is being used.  The user has to manually set the font name/path before they're allowed to modify the size/aa settings.  This issue is completely not listed in the help text at all.

When OpenTTD loads the default font it does so by calling "TryLoadDefaultTrueTypeFont" which will search the "baseset" directory.  "baseset" is the home directory for the OpenTTD-* font files. If you try:
"font medium OpenTTD-Sans 16 aa"
It will fail.  Calling SetFont which is what the console command code does will not search the baseset directory.  SetFont uses a different mechanism for loading the font file but, one that does not search the baseset directory.

You can use the generic name of a system font since OpenTTD has some code for dealing with Linux/Windows/etc... system fonts.  In that case a font name will suffice.

So depending on the use-case a font name might work. Unless it's an OpenTTD font in which case you will need to use the absolute path. 

Honestly the font command is a mess. It might be better just to rework the actual code and then update the docs on a much saner command.

Reply:
Okay I had to re-read the patch because my memory of it was totally off. I think what you have added is fine.

Yeah that person is me. :P I have a different discord display name.

It may be because I'm an idiot but, my first attempt to use this command I thought "name" was supposed to be a nickname. As if the command wanted me to name my new font settings.  Using "myfont" got me no where. Took me a minute to realize "name" meant name of a font. I'd like to see "name" to be "font name/path" but I can't/won't stop you over this.